snapcraft.yaml

```diff
@@ -1016,7 +1016,7 @@ parts:
   sqlite:
     source: https://github.com/sqlite/sqlite
     source-depth: 1
-    source-tag: version-3.44.2
+    source-tag: version-3.45.1
```

Comment:
I believe this depends on https://github.com/canonical/dqlite/pull/564

Reply:
I concur as SQLITE_DIRECT_OVERFLOW_READ was made default with sqlite 3.45.0